README.md

```diff
@@ -38,79 +34,9 @@ Results showed that the FOV-aware planner reduced interruptions and redundant ac
 </p>
 
 ## Installation Instructions ## 
-
-### Versions ###
-This project has mainly two versions:
-- `v1.0` is the version used for the VR demonstration presented in the 2025 ICRA submission paper.
-- `v2.2` is the latest version which is identical to `v1.0` accept the code has been refactored in some places and 
-  includes this readme with the instructions.
-For all the tags available please check the releases page associated with this project [here](https://github.com/SophieHsu/3d-plan-eval/tags).
-
-### Clone the Project ###
-Cloning the version of the code you want to run.
-```
-git clone git@github.com:SophieHsu/3d-plan-eval.git -b <version>
-conda create -y -n igibson python=3.8
-conda activate igibson
-pip install -e .
-pip install pygame toml
-```
-Download the [data](https://drive.google.com/file/d/1tVj27p_H2acWC53tl-BdVRsnJfH-0-XX/view?usp=sharing) and unzip it under the `igibson` folder.
-
-### Setup the Planner Project Repo ###
-Install the planner outside of this project folder. Link to the FOV-Aware planner repo: [link](https://github.com/SophieHsu/FOV-aware-planner)
-```
-git clone git@github.com:SophieHsu/FOV-aware-planner.git
-```
-Follow along the instructions in the [README](https://github.com/SophieHsu/FOV-aware-planner/blob/main/README.md) to setup this repo.
-
-
-### Install SteamVR ###
-Install the SteamVR (found [here](https://store.steampowered.com/app/250820/SteamVR/)) using the steam platform.
-
-
-### Install Oculus ###
-- Perform the initial setup of your VR headset. For oculus/quest see [this](https://www.meta.com/blog/quest/you-got-a-quest-2-heres-how-to-set-it-up/).
-- Set the VR headset to run with SteamVR. For oculus/quest see [this](https://docs.varwin.com/latest/en/instructions-for-using-the-oculus-quest-2-headset-2260861409.html).
-## Usage ##
-- Setup
-  - Start by setting up the  config and maps to be the same for both igibson and overcooked. If you're using the
-    provided config files. This has already been done for you. 
-- Starting the overcooked server
-  - Begin running the Overcooked server by navigating to the FOV-aware-planner and run the following script (a baked in restart mechanism for up to 5 restarts in case of crashes due to dropped connections)
```

Comment:
I think we should still keep some parts of these instructions. Also, definitely keep the restart mechanism.